src/glyphs.cpp

```diff
@@ -548,10 +548,6 @@ void RenderSDF(glyph_info &glyph,
         return;
     }
 
-    FT_Glyph ft_glyph = nullptr;
-    ft_glyph_guard glyph_guard(&ft_glyph);
```

Comment:
If this is not necessary should we remove the `ft_glyph_guard` struct as well? https://github.com/mapbox/node-fontnik/blob/f36bab5e4bdfc8257ba578056da423921ce03176/src/glyphs.cpp#L183-L193